spec/dummy_controller_spec.rb

```diff
@@ -373,5 +373,25 @@
         expect(response_body(response)['items']).to eq expected_list
       end
     end
+
+    context 'when paginating an ActiveRecord with a group by query' do
+      let!(:dummy_models) { create_list(:dummy_model, 12, name: 'argentina') }
+      let!(:dummy_models_2) { create_list(:dummy_model, 10, name: 'uruguay') }
+      let!(:dummy_models_3) { create_list(:dummy_model, 18, name: 'costa rica') }
+
+      let(:limit) { 2 }
+
+      before do
+        get :index_group_by, params: { per: limit }
```

Comment:
You should describe this method, and create a subject variable with this request
```rb
describe '#index_group_by' do
  subject(:http_request) { get :index_group_by, params: { per: limit } }

  context 'when paginating an ActiveRecord with a group by query' do
    # Here your lets and befores
end
```

Reply:
All tests was made like this :cry: